stories/topics/proc-aws-generate-restore-data-file.adoc

```diff
@@ -2,16 +2,23 @@
 
 = Generating the restore data file
 
-The following commands create a directory, and populate it with an empty data template that, when completed, is used during the restore. 
+The following commands create a directory, and populate it with an empty data template that, when completed, is used during the restore.
 
 .Procedure
-. Create a folder to hold the configuration 
+. Create a folder to hold the configuration
 +
 [literal, options="nowrap" subs="+attributes"]
 ----
 $ mkdir command_generator_data
 ----
 . Populate the `$(pwd)/command_generator_data` folder with the configuration file template.
++
+
+[NOTE]
+====
+On Linux, any file or directory created by the command generator will be owned by `root:root` by default. To change the ownership of the files and directories, you can run the `sudo chmod` command after the files are created. Check the xref:tech-note-linux-files-owned-by-root[tech notes] for more information.
```

Comment:
generator is owned by....

For more information, read the xref:tech-note-linux-files-owned-by-root[tech notes].